tests/paddle/test_paddle_model_export.py

```diff
@@ -295,3 +295,129 @@ def test_model_log_without_specified_conda_env_uses_default_env_with_expected_de
         conda_env = yaml.safe_load(f)
 
     assert conda_env == mlflow.paddle.get_default_conda_env()
+
+
+@pytest.fixture(scope="session")
+def get_dataset_built_in_high_level_api():
+    train_dataset = paddle.text.datasets.UCIHousing(mode="train")
+    eval_dataset = paddle.text.datasets.UCIHousing(mode="test")
+    return train_dataset, eval_dataset
+
+
+@pytest.fixture
+def pd_model_built_in_high_level_api():
+    class UCIHousing(paddle.nn.Layer):
+        def __init__(self):
+            super(UCIHousing, self).__init__()
+            self.fc_ = paddle.nn.Linear(13, 1, None)
+
+        def forward(self, inputs):  # pylint: disable=arguments-differ
+            pred = self.fc_(inputs)
+            return pred
+
+    train_dataset, test_dataset = get_dataset_built_in_high_level_api()
+
+    model = paddle.Model(UCIHousing())
+    optim = paddle.optimizer.Adam(learning_rate=0.01, parameters=model.parameters())
+    model.prepare(optim, paddle.nn.MSELoss())
+
+    model.fit(train_dataset, epochs=6, batch_size=8, verbose=1)
+
+    return ModelWithData(model=model, inference_dataframe=test_dataset)
+
+
+@pytest.mark.large
+def test_model_save_load_built_in_high_level_api(pd_model_built_in_high_level_api, model_path):
+    model = pd_model_built_in_high_level_api.model
+    test_dataset = pd_model_built_in_high_level_api.inference_dataframe
+    mlflow.paddle.save_model(pd_model=model, path=model_path)
+
+    reloaded_pd_model = mlflow.paddle.load_model(model_uri=model_path)
+    reloaded_pyfunc = pyfunc.load_pyfunc(model_uri=model_path)
+
+    low_level_test_dataset = []
+    for line in test_dataset:
+        data, _ = line
+        low_level_test_dataset.append(data)
```

Comment:
Can this be simplified to:

```
low_level_test_dataset = [x[0] for x in test_dataset]
```

Reply:
Yes! It's much better! Thank you for your suggestion